### PR TITLE
feat: updated ui5 maintenance versions

### PR DESCRIPTION
**Pull request by vadson71**
Adaptation to the latest UI5 maintenance changes

### CODE REVIEW

File: packages/semantic-model/src/utils.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export function error(message: string, throwError: boolean): void {
   if (throwError) {
     throw new Error(message);
   } else {
-    console.error(message);
+    // console.error(message);
```
Comment (voicis) [inline packages/semantic-model/src/utils.ts:23]: Should this be commented out?

File: packages/semantic-model/test/unit/api-negative.test.ts [hunk 2]
```diff
@@ -17,12 +17,12 @@ describe("The ui5-language-assistant semantic model package API negative tests",
     ): () => UI5SemanticModel {
       const generateCall = (): UI5SemanticModel => {
         return generate({
-          version: "1.71.49",
+          version: "1.71.61",
```
Comment (voicis) [inline packages/semantic-model/test/unit/api-negative.test.ts:20]: Wouldn't it be better to have a constant for the version here as well?

Comment (vadson71) [reply packages/semantic-model/test/unit/api-negative.test.ts]: Created a constant

File: packages/semantic-model/src/utils.ts (hunk 1 above)
Comment (vadson71) [reply packages/semantic-model/src/utils.ts]: reverted

**Review by mr25mr — APPROVED** [review]
LGTM

**Review by voicis — APPROVED** [review]

**Pull request merged**